src/main/features/offer/routes/defendant-response.ts

```diff
@@ -0,0 +1,44 @@
+import * as express from 'express'
+import { FormValidator } from 'forms/validation/formValidator'
+import { Paths } from 'offer/paths'
+import { Form } from 'forms/form'
+import { DefendantResponse } from 'offer/form/models/defendantResponse'
+import { ErrorHandling } from 'common/errorHandling'
+import User from 'idam/user'
+import { OfferGuard } from 'offer/guards/offerGuard'
+
+function getOffer (claim: any): any {
+  if (!claim.settlement || !claim.settlement.partyStatements) {
+    return undefined
+  }
+  return claim.settlement.partyStatements[0].offer
+}
+async function renderView (
+  form: Form<DefendantResponse>, res: express.Response, next: express.NextFunction) {
+  res.render(Paths.defendantResponsePage.associatedView, {
+    form: form,
+    claim : res.locals.user.claim,
+    offer: getOffer(res.locals.user.claim)
+  })
+}
+
+export default express.Router()
+  .get(
+    Paths.defendantResponsePage.uri,
+    OfferGuard.requestHandler,
```

Comment:
It does not make sense to have it here since guard redirects to conformation page. See below logic extracted from the guard.

```
logger.warn(`Claim ${claim.claimNumber} not eligible for an offer - redirecting to confirmation page`)
res.redirect(Paths.confirmationPage.evaluateUri({ externalId: user.claim.externalId }))
```

Please also use `GuardFactory.create` in the guard so that we have single class to mock and common mechanism of writing guards.

Reply:
This has not been addressed yet!!!